index-config/src/main/solr-config/uniprot-collections/taxonomy/conf/schema.xml

```diff
@@ -32,6 +32,8 @@
 	<field name="mnemonic" type="basic_ci" indexed="true" />
 	<field name="strain" type="full_ci" indexed="true"
 		multiValued="true" />
+	<field name="other_name" type="full_ci" indexed="true"
+		multiValued="true" />
```

Comment:
If other name can have numbers in it, it would be better to use the other solr type with string and number split.